terraform/cloudfront.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_cloudfront_distribution" "loris" {
+  origin {
+    domain_name = "${module.api_alb.dns_name}"
+    origin_id   = "loris"
+
+    custom_origin_config {
+      https_port = 443
+      http_port = 80
+      origin_protocol_policy = "match-viewer"
+      origin_ssl_protocols = ["TLSv1.2"]
```

Comment:
The format job should be whinging about this, but it doesn't notice changes in the `terraform` directory. Oops.